kotskinds/apis/kots/v1beta1/identity_types.go

```diff
@@ -17,71 +17,45 @@ limitations under the License.
 package v1beta1
 
 import (
-	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	runtime "k8s.io/apimachinery/pkg/runtime"
 )
 
-type IdentityConfigSpec struct {
-	Enabled                bool              `json:"enabled" yaml:"enabled"`
-	DisablePasswordAuth    bool              `json:"disablePasswordAuth,omitempty" yaml:"disablePasswordAuth,omitempty"`
-	Groups                 []IdentityGroup   `json:"groups,omitempty" yaml:"groups,omitempty"`
-	IngressConfig          IngressConfigSpec `json:"ingressConfig,omitempty" yaml:"ingressConfig,omitempty"`
-	AdminConsoleAddress    string            `json:"adminConsoleAddress,omitempty" yaml:"adminConsoleAddress,omitempty"`
-	IdentityServiceAddress string            `json:"identityServiceAddress,omitempty" yaml:"identityServiceAddress,omitempty"`
-	CACertPemBase64        string            `json:"caCertPemBase64,omitempty" yaml:"caCertPemBase64,omitempty"`
-	InsecureSkipTLSVerify  bool              `json:"insecureSkipTLSVerify,omitempty" yaml:"insecureSkipTLSVerify,omitempty"`
-	DexConnectors          DexConnectors     `json:"dexConnectors,omitempty" yaml:"dexConnectors,omitempty"`
+type IdentitySpec struct {
+	OIDCRedirectURLS            []string `json:"oidcRedirectURLS,omitempty" yaml:"oidcRedirectURLS,omitempty"`
```

Comment:
i think this is required... maybe remove omitempty

Reply:
good point

Reply:
actually, what if the vendor decides to not support oidc?

Reply:
you mean not support the identity service? they will omit the doc entirely